packages/playwright/src/createResourceArchive.ts

```diff
@@ -35,15 +35,15 @@ const idle = async (page: Page, networkTimeoutMs = DEFAULT_GLOBAL_RESOURCE_ARCHI
 export const createResourceArchive = async ({
   page,
   networkTimeout,
-  allowedArchiveDomains,
+  assetDomains,
 }: {
   page: Page;
   networkTimeout?: number;
-  allowedArchiveDomains?: string[];
+  assetDomains?: string[];
 }): Promise<() => Promise<ResourceArchive>> => {
   const cdpClient = await page.context().newCDPSession(page);
 
-  const watcher = new Watcher(cdpClient, allowedArchiveDomains);
+  const watcher = new Watcher(cdpClient, assetDomains);
```

Comment:
How about we make this same change in the [resource-archiver](https://github.com/chromaui/test-archiver/blob/main/packages/shared/src/resource-archive/index.ts#L34) (change `this.allowedArchiveOrigins` to `this.assetDomains` or something)?

Reply:
Hey! Just fixed this!